sdks/python/apache_beam/transforms/combiners_test.py

```diff
@@ -393,6 +398,18 @@ def test_global_fanout(self):
           | beam.CombineGlobally(combine.MeanCombineFn()).with_fanout(11))
       assert_that(result, equal_to([49.5]))
 
+  @attr('ValidatesRunner')
+  def test_hot_key_combining_with_accumulation_mode(self):
+    with TestPipeline() as p:
+      result = (p
+                | beam.Create([1, 2, 3, 4, 5])
+                | beam.WindowInto(GlobalWindows(),
+                                  trigger=Repeatedly(AfterCount(1)),
+                                  accumulation_mode=
+                                  AccumulationMode.ACCUMULATING)
+                | beam.CombineGlobally(sum).without_defaults().with_fanout(2))
```

Comment:
Why specify without_defaults()?

Reply:
It is really unnecessary. I followed the Java parity.

Reply:
If delete without_defaults(), we get this error:
ValueError: PCollection of size 2 with more than one element accessed as a singleton view. First two elements encountered are "1", "3". [while running 'CombineGlobally(sum, fanout=2)/InjectDefault']